.golangci.yaml

```diff
@@ -149,9 +149,9 @@ linters:
     - reassign
     - revive
     - rowserrcheck
-    - scopelint
+    # - scopelint
```

Comment:
This is replaced by exportloopref and exportloopref is already enabled.

Reply:
ack! Although https://github.com/kyoh86/looppointer seems like a more "nervous" replacement, but not readily available with golang-ci tools to my reading.